core/lib/presentation/utils/html_transformer/text/sanitize_html_transformers.dart

```diff
@@ -0,0 +1,17 @@
+
+import 'dart:convert';
+
+import 'package:core/presentation/utils/html_transformer/base/text_transformer.dart';
+import 'package:core/utils/app_logger.dart';
+
+class SanitizeHtmlTransformers extends TextTransformer {
+
+  const SanitizeHtmlTransformers();
+
+  @override
+  String process(String text) {
+    final htmlEncoded = const HtmlEscape().convert(text);
+    log('HtmlSanitizerTransformers::process:htmlEncoded: $htmlEncoded');
```

Comment:
Is this log necessary?

Reply:
dont worry, this log is handled by `build flag`

<img width="558" alt="image" src="https://github.com/linagora/tmail-flutter/assets/6462404/cac046fb-c013-47fe-8023-421356dd8b71">